tox.ini

```diff
@@ -15,13 +15,14 @@ DJANGO_SETTINGS_MODULE = apptemplates.test.settings
 [tox]
 envlist =
     flake8-py{26,27,32,33,34,35}
-    pylint-py{27,35}-django110
+    pylint-py{27,35}-django{110,111}
```

Comment:
I think it's sufficient to stick to the latest version, e.g.
``` bash
pylint-py{27,36}-django111
```
This declaration serves only to run Pylint against version 2 and 3 of Python (with any, e.g. the latest, Django version installed).